modules/options.js

```diff
@@ -230,9 +244,14 @@ export class StreamViewOptions {
 			scope: 'world',
 			config: true,
 			restricted: true,
-			requiresReload: true,
-			default: true,
-			type: Boolean,
+			requiresReload: false,
+			choices: {
+				[StreamViewOptions.ChatVisibility.NEVER]: StreamViewOptions.localizeChatVisibility(StreamViewOptions.ChatVisibility.NEVER),
+				[StreamViewOptions.ChatVisibility.ALWAYS]: StreamViewOptions.localizeChatVisibility(StreamViewOptions.ChatVisibility.ALWAYS),
+				[StreamViewOptions.ChatVisibility.ENCOUNTER]: StreamViewOptions.localizeChatVisibility(StreamViewOptions.ChatVisibility.ENCOUNTER),
+			},
+			default: StreamViewOptions.PreviewDisplay.LAYER,
```

Comment:
Copy-pasta, probably should be `StreamViewOptions.ChatVisibility.ALWAYS`

Reply:
good catch :)